src/components/Card/Card.stories.js

```diff
@@ -5,5 +5,9 @@ export default {
   component: Card,
 }
 
-export const defaultCard = () =>
-  Card('FirstName LastName', 'https://source.unsplash.com/random/500x500')
+const person = {
```

Comment:
I think in your Card component you also use the house to get the house-color. Maybe you should display this in storybook as well?

Reply:
ah true! forgot it here 😅 thanks!